woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -132,6 +132,36 @@ def test_accessor_schema_property(sample_df):
     assert sample_df.ww._schema == sample_df.ww.schema
 
 
+def test_set_accessor_name(sample_df):
+    df = sample_df.copy()
+    error = re.escape("Woodwork not initialized for this DataFrame. Initialize by calling DataFrame.ww.init")
+    with pytest.raises(WoodworkNotInitError, match=error):
+        df.ww.name
+    with pytest.raises(WoodworkNotInitError, match=error):
+        df.ww.name = 'name'
+
+    df.ww.init()
```

Comment:
Can we also add a test case where we init woodwork on a DataFrame with a name and then change it to something else?

Reply:
done!